### DISCUSSION TITLE
Item rarity

### DISCUSSION THREAD

**Discussion by iollmann** [kinkofer/FightClub5eXML#185] [Q&A]
I'm trying to figure out how FC5 assigns item rarity based on the compendium.  For example, in the Ring of Jumping, FC5 says "Ring, uncommon (requires attunement)"  directly below the name.  In the xml, I don't see any tags for this information other than 

\<type\>RG\<\/type\>

or maybe 

\<detail\>major\<\/detail\>

I'm assuming the RG type just means ring.  Am I supposed to infer uncommon and attunement out of "major"?

`  <item> `
`      <name>Ring of Jumping</name>`
`    <detail>major</detail> `
`    <type>RG</type> `
`    <weight/> `
`    <property/> `
`    <text>While wearing this ring, you can cast the jump spell from it as a bonus action at will, but can target only yourself when you do so. `
`Found On: Magic Item Table F `
`Source: Dungeon Master's Guide p. 191</text> `
`    </item> `

**Comment by kinkofer** [comment]
<detail> is where this information would go, but the source I'm using to compile all the data had "major" for these values. I'll go back eventually to update the data from older sources and hopefully it will be more specific.

**Reply by iollmann** [reply]
It looks like the first word of your reply is missing (on my browser?)  I'm assuming this is because it is a \<tag\>.  Maybe some \\ backslashes would help? I suppose we should infer that the tag is \<detail\>?

Thanks for the update!  It will be valuable.

**Reply by kinkofer** [reply]
Sorry, yes, `<detail>` is what I was saying. Didn't realize the comment would parse the detail tag.

**Reply by iollmann** [reply]
For those playing the home game, XML and HTML are quite similar, so unfortunately, when we talk about a XML tag such as \<item\>, it will probably get interpreted to be a HTML \<tag\> and may be removed by the browser when it reads it.  This can be avoided by using \\ backslashes as escape characters in front of misinterprettable characters, such as \\\<tag\\\> so that they appear as intended.  Indeed, the backslash itself can be escaped so that it can be seen by using a double backslash \\\\.  

Meta: you can only imagine what my original text looked like here with  \\\\\\\<tag\\\\\\\> for \\\<tag\\\>  and `\\\\\\\<tag\\\\\\\>` for `\\\<tag\\\>`. 

kinkofer uses the code tag \`\<detail\>\` with single lefthand quotes above, which is another alternative that can avoid excessive backslashes. It will cause the text to also acquire a `grey square` around it.

**Reply by kinkofer** [reply]
If you use back ticks (`) around your code you don't have to add backslashes
